CMakeLists.txt

```diff
@@ -48,6 +48,20 @@ else()
   message(STATUS "mvIMPACT NOT found => BlueFox2 support disabled")
 endif()
 
+INCLUDE_DIRECTORIES("/usr/include/flycapture")
```

Comment:
The FlyCap library detection should be moved to a CMake find module, please see [cmake/FindmvIMPACT.cmake](https://github.com/RoboCup-SSL/ssl-vision/blob/ecd668a8d5f1afc416f05aad34be4cb331e9244f/cmake/FindmvIMPACT.cmake) for an example, and [CMakeLists.txt:41](https://github.com/RoboCup-SSL/ssl-vision/blob/ecd668a8d5f1afc416f05aad34be4cb331e9244f/CMakeLists.txt#L41) to see how to use it.